components/backends/vllm/launch/agg.sh

```diff
@@ -8,4 +8,4 @@ trap 'echo Cleaning up...; kill 0' EXIT
 python -m dynamo.frontend &
 
 # run worker
-python -m dynamo.vllm  --model Qwen/Qwen3-0.6B
+python -m dynamo.vllm --model Qwen/Qwen3-0.6B --enforce-eager --no-enable-prefix-caching
```

Comment:
Why adding `--enforce-eager` and `--no-enable-prefix-caching`? Same question for below